flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/slotmanager/FineGrainedSlotManagerTestBase.java

```diff
@@ -142,7 +142,7 @@ static void assertFutureNotComplete(CompletableFuture<?> completableFuture) thro
         private final ScheduledExecutor scheduledExecutor = TestingUtils.defaultScheduledExecutor();
         private final Executor mainThreadExecutor = MAIN_THREAD_EXECUTOR;
         private FineGrainedSlotManager slotManager;
-        private long requirementCheckDelay = 0;
+        private long requirementCheckDelay = 20;
```

Comment:
We might need to find the root cause instead of simply increasing the delay. IIUC, even though the delay is `0`, the test should not fail.